widgets/Html/index.jsx

```diff
@@ -66,16 +68,16 @@ class Html extends Component {
   /**
    * If the user tapped a link element, prevent the default behaviour
    * and handle them via ParsedLink.
-   * @param {Object} event The touchstart event.
+   * @param {Object} e The touchstart event.
    */
-  handleTap = (event) => {
-    const aTag = event.target.closest('a');
+  handleTap = (e) => {
```

Comment:
Why do this?

Reply:
I've imported 'event' before in here. Then eslint complaint. Forgot to revert.